openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -145,31 +145,31 @@ instances:
     #
     # extra_metrics: []
 
-    ## @param exclude_metrics - list of strings - optional
```

Comment:
I believe that `exclude_metrics` is the name of this config in OpenMetricsV2 while `ignore_metrics` is the name for OpenMetricsV1. You can see that `exclude_metrics` is used [here](https://github.com/DataDog/integrations-core/blob/71b041eb30c98e6b6a3149a9eb6f0d039d676877/datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/scraper.py#L101) in the V2 check. 

This `conf.yaml.example` is only showing the V2 config options because V1 is now deprecated. Most of the time, when we refer to OpenMetrics it will be referring to V2, except for the links you have in the PR description which reference V1.